### PR TITLE
Improvement for en/docs/reference/glossary/api-eviction.md

### LINKED ISSUE #32935: Improvement for en/docs/reference/glossary/api-eviction.md
**This is a Feature Request**

**What would you like to be added**

I think it is difficult to understand the difference between this and the DELETE API.
The Eviction API complies with PodDisruptionBudget.
Therefore, I would like to add new lines that explain about PodDisruptionBudget like this.
```
API-initiated evictions respect your configured [`PodDisruptionBudgets`](/docs/tasks/run-application/configure-pdb/)
and [`terminationGracePeriodSeconds`](/docs/concepts/workloads/pods/pod-lifecycle#pod-termination).
```
(This is from [this page](https://kubernetes.io/docs/concepts/scheduling-eviction/api-eviction/))

### PR DESCRIPTION
**Pull request by 196Ikuchil**
Resolves #32935 
I think that it is also important to explain about PodDisruptionBudget.
Therefore, I added both.

### CODE REVIEW

File: content/en/docs/reference/glossary/api-eviction.md [hunk 1]
```diff
@@ -19,4 +19,9 @@ You can request eviction either by directly calling the Eviction API
 using a client of the kube-apiserver, like the `kubectl drain` command. 
 When an `Eviction` object is created, the API server terminates the Pod. 
 
+API-initiated evictions respect your configured [`PodDisruptionBudgets`](/docs/tasks/run-application/configure-pdb/)
+and [`terminationGracePeriodSeconds`](/docs/concepts/workloads/pods/pod-lifecycle#pod-termination).
+
 API-initiated eviction is not the same as [node-pressure eviction](/docs/concepts/scheduling-eviction/eviction/#kubelet-eviction).
+
+* See [API-initiated eviction](/docs/concepts/scheduling-eviction/api-eviction/) for more information.
```
Comment (sftim) [inline content/en/docs/reference/glossary/api-eviction.md:27]:
```suggestion
See [API-initiated eviction](/docs/concepts/scheduling-eviction/api-eviction/) for more information.
```

**Review by sftim — COMMENTED** [review]
/lgtm

Thanks

**Comment by riita10069** [comment]
Looks like that PR has been merged, so let's reflect that here.

**Pull request merged**